INSTALL.rst

```diff
@@ -11,24 +11,39 @@ Prerequisite Software
 ---------------------
 
 HaaS requires a number of packages available through the CentOS RPM
-repositories, as well as the EPEL repository. EPEL can be enabled via:
-
-::
+repositories, as well as the EPEL repository. EPEL can be enabled via::
 
     yum install epel-release
 
-Then, the rest of the packages can be installed via:
-
-::
+Then, the rest of the packages can be installed via::
 
-    yum install libvirt bridge-utils ipmitool telnet sqlite httpd mod_wsgi python-pip qemu-kvm python-virtinst
+    yum install libvirt bridge-utils ipmitool telnet httpd mod_wsgi python-pip qemu-kvm python-virtinst virt-install python-psycopg2
```

Comment:
Good catch. Eventually I'd like us to be doing everything (including haas itself) via distro packages, but right now installing it via pip would be more consistent with how we're handling the rest of the python dependencies. We should definitely remove one of these though.